misc/genstd/testdata/linkFunctions_TypedValue/std/std.gno

```diff
@@ -6,6 +6,6 @@ type Banker interface {
 
 func TVParam(m struct{ m1 map[string]interface{} })
 
-func TVResult() interface{ S() map[int]Banker }
+func TVResult() interface{}
```

Comment:
This change seems to be completely unrelated to the rest of the PR. Why this?

Reply:
It's really to use this `TVResult()` to test `interface{}` mapping.

In reality, `gno.X` isn't able to handle `interface{}` type expressions, so it's a better usage of this test case to try to convert a `interface{}` result rather than a `interface{ ... }` result which would never fly anyway in the real scenario, that was the logic.